x-pack/legacy/plugins/graph/public/components/guidance_panel/guidance_panel.tsx

```diff
@@ -29,6 +28,7 @@ import {
 import { IndexPatternSavedObject } from '../../types';
 import { openSourceModal } from '../../services/source_modal';
 
+import { IDataPluginServices } from '../../../../../../../src/plugins/data/public';
```

Comment:
nit: this could be `import { IDataPluginServices } from 'src/plugins/data/public'`

Reply:
fixed